packages/terra-embedded-content-consumer/tests/jest/EmbeddedContentConsumer.test.jsx

```diff
@@ -8,7 +8,7 @@ it('should render the embedded content consumer container', () => {
     src="https://www.google.com/"
   />);
 
-  const wrapper = shallow(embeddedContentConsumer);
```

Comment:
What's the reason we're switching from shallow rendering to rendering the static html?

Reply:
The event listeners in the EmbeddedContentConsumer component were throwing errors after updating to enzyme v3 using the `shallow` method. My understanding on why this is an issue now is a bit rough but here's what I'm thinking. Attaching an event listener to the document with `addEventListener` means the event is not being handled by React's synthetic event system and I believe enyzme's `shallow` and `mount` methods use React's synthetic event system while `render` uses another implementation which supports `addEventListener`.